package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/rtfeldman/node-elm-test#readme",
   "dependencies": {
+    "fs-extra": "^0.23.1",
```

Comment:
The correct thing appears to be `~0.23.1`, which will match 0.23.*  http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json